doku/tests/printers/array/of_type/enum/of_tag/adjacent/of_fields/named/output.toml

```diff
@@ -0,0 +1,14 @@
+[[inner]]
```

Comment:
It looks like the `toml` crate prints it a bit differently (https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=30057d7ba26e0eb3cb59c25686717a5d) 🐛